plugin/file/secondary.go

```diff
@@ -160,23 +165,32 @@ Restart:
 
 		case <-refreshTicker.C:
 
+			start = time.Now()
+			RefreshSeconds.WithLabelValues(z.origin).Observe(time.Since(start).Seconds())
```

Comment:
this just sets it to 0? Or a value close to it?

Reply:
Yes, this observation doesn't make any sense, since there's nothing between the `Now()` and the observation.